internal/controller/streaming_manager.go

```diff
@@ -48,7 +48,9 @@ type EnvironmentConfig struct {
 	ClusterCreds string
 }
 
-var resourceTypes = [2]string{"pods", "nodes"}
+var resourceTypes = []string{"pods", "nodes", "serviceaccounts", "replicationcontrollers", "replicasets", "deployments", "statefulsets",
```

Comment:
This is a recipe for bugs. Don't duplicate the list of supported resource kinds in two different places.
This list should be specified only once.